lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -508,26 +519,11 @@ function checkIp(value) {
         // we would accept different ip formats, which is not
         // standard in an AWS use case.
         try {
-            try {
-                parseCIDR(values[i]);
-            } catch (err) {
-                isValid(values[i]);
-            }
+            parseCIDR(values[i]);
```

Comment:
Check the comment on the commit.

This was running but any error was completely discarded.
Based on the comment, shouldn't we drop the parsing from ipaddr.js as we have stricted regex ?